Tests/MeshLib/TestMeshValidation.cpp

```diff
@@ -46,9 +46,9 @@ detectHoles(MeshLib::Mesh const& mesh,
 TEST(MeshValidation, DetectHolesTri)
 {
     std::array<double, 12> pix = {{0,0.1,0.2,0.1,0,0,0.1,0,0,0,-0.1,0}};
-    GeoLib::RasterHeader const header = {
+    GeoLib::RasterHeader header = {
         4, 3, 1, MathLib::Point3d(std::array<double, 3>{{0, 0, 0}}), 1, -9999};
-    GeoLib::Raster const raster(header ,pix.begin(), pix.end());
+    GeoLib::Raster const raster(std::move(header), pix.begin(), pix.end());
```

Comment:
I'd agree here if RasterHeader would have a move ctor. But it is a simple aggregate, which have to be copied anyway.